test/conformance/tests/sriov_operator.go

```diff
@@ -1093,43 +1093,86 @@ var _ = Describe("[sriov] operator", func() {
 
 			Context("MTU", func() {
 				BeforeEach(func() {
-					node := sriovInfos.Nodes[0]
-					intf, err := sriovInfos.FindOneSriovDevice(node)
-					Expect(err).ToNot(HaveOccurred())
+					var node string
+					resourceName := "mturesource"
+					numVfs := 5
+					var intf *sriovv1.InterfaceExt
+					var err error
 
-					mtuPolicy := &sriovv1.SriovNetworkNodePolicy{
-						ObjectMeta: metav1.ObjectMeta{
-							GenerateName: "test-mtupolicy",
-							Namespace:    operatorNamespace,
-						},
+					if discovery.Enabled() {
+						node, resourceName, numVfs, intf, err = discovery.DiscoveredResources(clients,
+							sriovInfos, operatorNamespace,
+							func(policy sriovv1.SriovNetworkNodePolicy) bool {
+								if !defaultFilterPolicy(policy) {
+									return false
+								}
+								if policy.Spec.Mtu != 9000 {
```

Comment:
Does Mtu need to be EXACTLY 9000?

Reply:
No
Afair, a jumbo frame is anything above 1500. Changing to > 1500